cmake/libprotoc.cmake

```diff
@@ -92,6 +91,7 @@ set(libprotoc_files
   ${protobuf_source_dir}/src/google/protobuf/compiler/ruby/ruby_generator.cc
   ${protobuf_source_dir}/src/google/protobuf/compiler/subprocess.cc
   ${protobuf_source_dir}/src/google/protobuf/compiler/zip_writer.cc
+  ${protobuf_BINARY_DIR}/well_known_types_embed.cc		# This file is generated, therefore is in different directory
```

Comment:
This file list here is generated by the update_file_lists.sh script:
https://github.com/google/protobuf/blob/master/update_file_lists.sh

So your change here won't stand unless you make a corresponding change to src/Makefile.am. Can you make a change to src/Makefile.am accordingly?

Reply:
I will try, although I have no experience with automake .. but for now I have

Reply:
something that could fix it for a few days